include/zephyr/net/ieee802154_radio.h

```diff
@@ -296,6 +311,12 @@ enum ieee802154_config_type {
 	 *  should disable it for all enabled addresses.
 	 */
 	IEEE802154_CONFIG_ENH_ACK_HEADER_IE,
+
+	/** Number of config values defined in ieee802154_config_type. */
```

Comment:
config values -> types (see the API's nomenclature)

Reply:
fixed